x-pack/plugins/log_explorer/public/components/common/hover_popover.tsx

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useRef, useState } from 'react';
+import { EuiPopover, EuiPopoverTitle } from '@elastic/eui';
+
+export const HoverPopover = ({
+  children,
+  button,
+  title,
+}: {
+  children: React.ReactChild;
+  button: React.ReactElement;
+  title: string;
+}) => {
+  const [isPopoverOpen, setIsPopoverOpen] = useState(false);
+  const leaveTimer = useRef<NodeJS.Timeout | null>(null);
+
+  const onMouseEnter = () => {
+    if (leaveTimer.current) {
+      clearTimeout(leaveTimer.current);
+    }
+    setIsPopoverOpen(true);
+  };
+
+  const onMouseLeave = () => {
+    leaveTimer.current = setTimeout(() => setIsPopoverOpen(false), 100);
+  };
```

Comment:
**question:** I've noticed we used this approach already a couple of times to make the popover open on hover.
Do you think is worth talking with the EUI team about implementing and testing this capability on the component? 
I think many others could benefit by this change.

Reply:
I did talk with the EUI team initially when i 1st implemented this. They proposed a complete native HTML CSS implementation to achieve this. But given we are using it more often now, i can ask them again

Reply:
I have started a thread with EUI team on Slack. As this is outside the scope of this ticket, will mark this as resolved for now